Automattic/wp-calypso
# Subscriptions Management: Add View feed button to each site subscription row

Resolves https://github.com/Automattic/wp-calypso/issues/82229

![V7NyNI.png](https://github.com/Automattic/wp-calypso/assets/2019970/2a13304f-ad5b-4c0f-86d4-43192dd4d650)

## Proposed Changes

* Add "View feed" button inside the subscriptions' `SiteSettingsPopover`
* The button has an href pointing to `/read/feed/$feedId`
* Add ellipsis menu toggle title ("More actions") for the `SiteSettingsPopover` for improved accessibility and testing
* Track the button click with the `calypso_subscriptions_view_feed_button_clicked` event name. 
* Change the order of the Unsubscribe button in the popover's layout
  * p1695820872992539/1695810424.438149-slack-C02TCEHP3HA

## Testing Instructions



* Go to `/read/subscriptions`
* Press the More actions button (3 dots icon) in the last cell of a site subscription row, to open up the subscription settings popover 
* Ensure the View feed button is visible and opens the expected URL

## Pre-merge Checklist
